src/app/modules/item/pages/item-chapter-view-group-situation/item-chapter-view-group-situation.component.ts

```diff
@@ -44,11 +52,23 @@ export class GroupSituationChapterViewComponent implements OnChanges, OnDestroy
 
   data: Data = {
     type: this.defaultFilter,
-    rowsHeader: [],
     items: [],
-    data: [],
+    rows: [],
+    can_access: false,
   }
 
+  permissions: Permissions = {
```

Comment:
For a reader, it is quite difficult to understand what is a non-array "permissions" instance variable on ItemChapterViewGroupSituationComponent. From what I understand, this is only the permissions you pass to the dialog. If so, call it "dialogPermissions".

Reply:
fixed